### PR TITLE
SCA Rocky Linux 9

### PR DESCRIPTION
**Pull request by Johnng007**
|Related issue|
|---|
|https://github.com/wazuh/wazuh/issues/16618|

- Compilation without warnings in every supported platform
  - [ ] Linux
  - [ ] Windows
  - [ ] MAC OS X
- [ ] Source installation
- [ ] Package installation
- [ ] Source upgrade
- [ ] Package upgrade
- [ ] Review logs syntax and correct language
- [ ] QA templates contemplate the added capabilities

- Memory tests for Linux
  - [ ] Scan-build report
  - [ ] Coverity
  - [ ] Valgrind (memcheck and descriptor leaks check)
  - [ ] Dr. Memory
  - [ ] AddressSanitizer
- Memory tests for Windows
  - [ ] Scan-build report
  - [ ] Coverity
  - [ ] Dr. Memory
- Memory tests for macOS
  - [ ] Scan-build report
  - [ ] Leaks
  - [ ] AddressSanitizer

- [ ] Retrocompatibility with older Wazuh versions
- [ ] Working on cluster environments
- [ ] Configuration on demand reports new parameters
- [ ] The data flow works as expected (agent-manager-api-app)
- [ ] Added unit tests (for new features)
- [ ] Stress test for affected components

- Decoder/Rule tests
  - [ ] Added unit testing files ".ini"
  - [ ] runtests.py executed without errors

### CODE REVIEW

**Review by 72nomada — APPROVED** [review]
LGTM

**Review by DFolchA — COMMENTED** [review]
The directory names do not conform with the information retrieved by `dist-detect.sh`:
```
[root@rockylinux9 vagrant]# bash dist-detect.sh 
DIST_NAME=rocky
DIST_VER=9
DIST_SUBVER=2
```

File: etc/templates/config/rocky/9/sca.files [hunk 1]
```diff
@@ -0,0 +1 @@
+rockylinux/cis_rocky_linux_9.yml
```
Comment (DFolchA) [inline etc/templates/config/rocky/9/sca.files:1]: This path is incorrect after the last changes.

**Review by DFolchA — CHANGES_REQUESTED** [review]
The new files must be added to `sca.manager.files`

**Review by ooniagbi — APPROVED** [review]
LGTM!

**Review by ooniagbi — APPROVED** [review]
LGTM!

**Pull request merged**